src/components/App.tsx

```diff
@@ -1628,13 +1638,62 @@ class App extends React.Component<AppProps, AppState> {
       textAlign: this.state.currentItemTextAlign,
       verticalAlign: DEFAULT_VERTICAL_ALIGN,
       locked: false,
-    });
+    };
+    const DELTA_GAP = 12;
+    const newElements: ExcalidrawTextElement[] = [];
+    if (splitText) {
+      let currentY = y;
+      const splittedText = text.split("\n");
+      for (const line of splittedText) {
+        if (line.trim().length === 0) {
```

Comment:
As written, this drops blank lines in pasted text (like for new paragraphs).  Perhaps add a line like `currentY += getApproxLineHeight(font);` before `continue;`?

Reply:
Yes, corrected. Thank you! :D